components/release/base/cronjobs/remove-expired-releases.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: remove-expired-releases
+  namespace: release-service
+spec:
+  schedule: "10 03 * * *"
+  successfulJobsHistoryLimit: 7
+  failedJobsHistoryLimit: 7
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: release-service-controller-manager
+          containers:
+            - name: release-cleanup
+              command:
+                - /bin/bash
+                - -c
+                - |
+                  set -o pipefail
+                  PATH="/bin:/usr/bin:/usr/local/bin"
+                  NOW=$(date +%s)
+                  kubectl get release --all-namespaces \
+                  --sort-by=.status.expirationTime \
+                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.expirationTime}}{{"\n"}}{{end}}' > /tmp/kubectl-out
```

Comment:
Why is `/tmp/kubectl-out` used directly and `EXPIRED_RELEASES_FILE` is defined as an env var? I would probably suggest defining both at the top of the script - below `NOW=`.

Reply:
Sure. Just thought the "eventually" we would like to put this stuff in some configmap. I will move it to the script.